test-bench-utils/README.md

```diff
@@ -33,21 +33,29 @@ user input, and for the `/safe` and `/noop` endpoints it is called with the
 ## Front-end content
 If there is any custom data you want to provide to the test bench front end, you
 can export it from _lib/content/_. For example, we export the following XML
-string as a potential attack for the xxe rule:
+string as a potential attack for the xpath injection rule:
 
-_lib/content/xxe.js_
+_lib/content/xpathInjection.js_
```

Comment:
was this changed because you took the constant attackXml out of this repo?

Reply:
Yea I removed that file, so figured the README shouldn't refer to a file that no longer exists.  This looked like another case where we were doing the same kind of thing.